plugins/docker_binary_builder/app/models/binary_builder.rb

```diff
@@ -0,0 +1,132 @@
+class BinaryBuilder
+  DOCKER_FILE = 'Dockerfile.build'.freeze
```

Comment:
Could you call this `DOCKER_BUILD_FILE` so it's more clear which Dockerfile it's referring to?  I was a little confused when I came across the constant on line 17.